product_docs/docs/slony/1.0/01_installation/index.mdx

```diff
@@ -22,14 +22,10 @@ The following table lists the latest Slony Replication versions and their corres
 | `Slony Replication 2.2`                 | EDB Postgres Advanced and PostgreSQL Server 10 and 9.6 | RHEL 7 - x86_64 <br/>RHEL 7 - ppc64le <br/>Windows x86_64 and Linux graphical installer                                                                                                     |
 
 !!! Note
-    Slony is not supported on CentOS/RHEL/OL 6.x platforms. EDB strongly recommends migrating all EDB products running on these platforms to a supported platform.
+    Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform.
```

Comment:
Don't change second sentence

Reply:
I have changed to maintain consistency as it is "Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform" in all guides.